internal/models/network.go

```diff
@@ -34,8 +34,11 @@ type CompressedPayload struct {
 }
 
 type Device struct {
-	RpiUptimeSecs  int     `json:"rpiUptimeSecs,omitempty"`
-	BatteryVoltage float64 `json:"batteryVoltage,omitempty"`
+	RpiUptimeSecs   int     `json:"rpiUptimeSecs,omitempty"`
+	BatteryVoltage  float64 `json:"batteryVoltage,omitempty"`
+	SoftwareVersion string  `json:"softwareVersion,omitempty"`
+	IMEI            string  `json:"imei,omitempty"`
+	UnitID          string  `json:"unitId,omitempty"`
```

Comment:
just thinking we should rename this to Serial, so more compatible when this payload comes from other devices (UnitID is autopi specific but we refer to it as serial everywhere)